### PR TITLE
client: add support dark colors via settings file (dark_mode variable)

### LINKED ISSUE #171: Feature Request: coloring via config
**Is your feature request related to a problem? Please describe.**
My terminal with the solarized-dark theme does not show me the correct green colors. 
colors

**Describe the solution you'd like**
It will be nice if the colors can change from the config file

### PR DESCRIPTION
**Pull request by Mephistophiles**
This commit adds the ability to choose  dark colors instead of regular colors . It may
was  useful  for solarized-like themes (green seems weird). By default
the client is using dark_mode = false.

Closes #171

### CODE REVIEW

**Comment by Mephistophiles** [comment]
After review I will create the PR to pueue-lib

File: client/display/colors.rs [hunk 1]
```diff
@@ -0,0 +1,42 @@
+use crossterm::style::Color;
+use pueue_lib::settings::Settings;
+
+pub struct Colors {
```
Comment (Nukesor) [inline client/display/colors.rs:4]: A few lines of comments what this class is for would be awesome.

**Review by Nukesor — CHANGES_REQUESTED** [review]
Nice!

Code looks good and clean.
One little nitpick, otherwise this looks good to me.

Feel free to open that PR for `pueue-lib`.
I'll then release a new patch version for this change, so you can properly set the dependency in this PR.

**Review by Nukesor — CHANGES_REQUESTED** [review]
Oh, two more things.

1. Could you already go ahead and pin the `pueue-lib` version to `0.12.1`?
That way there's no way to get some non-compiling version as soon as 0.12.1 is released.

2. You should add that change to the changelog! Don't forget to mention yourself and to link your Github profile :)

**Comment by Mephistophiles** [comment]
> Could you already go ahead and pin the pueue-lib version to 0.12.1?
That way there's no way to get some non-compiling version as soon as 0.12.1 is released.

Fixed, I will update Cargo.lock file after pueue-lib is released.

> You should add that change to the changelog! Don't forget to mention yourself and to link your Github profile :)

Should I add the changelog notes to the  `0.12.0` release cycle?

**Comment by Nukesor** [comment]
1. Ah, of course. I completely forgot about the Cargo.lock :D
Pueue-lib v0.12.1 is released, you should now be able to properly lock the version. Sorry for any inconveniences.

2. Jep, the v0.12 release cycle is the correct place.
I'm planning to release v0.12 in the next few days :)

**Comment by Mephistophiles** [comment]
Cargo.lock and Changelog have been updated Thanks for your review ! :)

**Comment by Nukesor** [comment]
Perfect! Code looks good to me.

Thanks for tackling this issue, much appreciated!

**Pull request merged**